CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Confluent's Golang client for Apache Kafka
 
+## v1.9.0
+
+This is a maintenance release:
+
+ * Fix Rebalance events behavior for static membership and fix for consumer close takes longer than 10s (@jliunyu, #757).
```

Comment:
Split this up into two entries, since they're two issues.
Rephrase the second one to:
" * Fix consumer close taking 10 seconds when there's no rebalance needed"